.github/workflows/macos-cmake.yml

```diff
@@ -103,6 +103,12 @@ jobs:
         fi
     - name: Install bash >= 5.x and Ninja
       run: brew install bash ninja
+    - name: Download cmake formula(3.27.2) and install
+      run: |
+        brew install wget
+        brew unlink cmake
+        wget https://raw.githubusercontent.com/Homebrew/homebrew-core/fd21fcf239bcd0231c9fed5719403ec128151af4/Formula/cmake.rb
```

Comment:
Should this have a "working-directory" that places the download somewhere temporary?

Reply:
I think so.  Let me merge this first, as it fixes the builds, and I will send a second PR to clean up things.